app/src/main/java/com/github/epfl/meili/photo/PhotoEditActivity.kt

```diff
@@ -137,39 +162,39 @@ class PhotoEditActivity : AppCompatActivity() {
     }
 
     private fun toggleDrawing() {
-        if (!photoEditor.brushDrawableMode)
-            startDrawing()
+        //start drawing, function combined because only 20 functions allowed by code climate
+        if (!photoEditor.brushDrawableMode) {
+            stopText()
+            stopFilters()
+            photoEditor.setBrushDrawingMode(true)
+            binding.paintModeButton.setBackgroundColor(getColor(R.color.quantum_bluegrey100))
+            binding.colorSlider.visibility = View.VISIBLE
+        }
         else
             stopDrawing()
     }
 
-    private fun startFilters() {
-        stopDrawing()
-        binding.filters.setBackgroundColor(getColor(R.color.quantum_bluegrey100))
-        binding.filtersContainer.visibility = View.VISIBLE
-    }
 
     private fun stopFilters() {
         binding.filters.setBackgroundColor(0)
         binding.filtersContainer.visibility = View.GONE
     }
 
     private fun toggleFilters() {
-        if (!binding.filtersContainer.isVisible)
-            startFilters()
+        if (!binding.filtersContainer.isVisible) {
+            //start filters
+            stopDrawing()
+            stopText()
+            binding.filters.setBackgroundColor(getColor(R.color.quantum_bluegrey100))
+            binding.filtersContainer.visibility = View.VISIBLE
+        }
         else
             stopFilters()
     }
 
-    /** Show scrollable emoji table */
-    private fun showEmojiTable() {
-        binding.emojiContainer.visibility = View.VISIBLE
-    }
-
-
-    private fun changeDrawingColor() {
-        photoEditor.brushColor = binding.colorSlider.color
-    }
+    //private fun changeDrawingColor() {
+    //    photoEditor.brushColor = binding.colorSlider.color
+    //}
```

Comment:
This is commented code, do we have to keep it here ?

Reply:
I thought maybe someone will want to reuse it, but anyway it's in git's history so we can remove it.